composer.json

```diff
@@ -90,7 +90,7 @@
         "symfony/web-profiler-bundle": "4.4.*",
         "symfony/yaml": "4.4.*",
         "twig/twig": "^2.14",
-        "willdurand/js-translation-bundle": "3.0.*"
+        "willdurand/js-translation-bundle": "^3.0|^4.0"
```

Comment:
Nothing fancy here - the only significant change is that support for symfony 3.4 and lower was removed. 
see https://github.com/willdurand/BazingaJsTranslationBundle/compare/3.0.1...4.0.1

Reply:
We can just keep the `^4.0`.